test/fixtures/users-schema.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+  name: 'users',
+  idField: 'id',
+  primaryKeys: ['id'],
+  isCompositePrimary: false,
+  fields: [
+    {
+      field: 'smart',
+      type: 'String',
+      get: () => ({ foo: 'bar' }),
+      isVirtual: true,
+      isFilterable: false,
+      isSortable: false,
+      isReadOnly: true,
+      defaultValue: null,
+      isRequired: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      validations: [],
+      integration: null,
+    },
+    {
+      field: 'id',
+      type: 'String',
+      columnName: 'id',
+      primaryKey: true,
+      isRequired: true,
+      validations: [Array],
+      defaultValue: null,
+      isReadOnly: false,
+      isSortable: true,
+      isFilterable: true,
+      isVirtual: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      integration: null,
+    },
+    {
+      field: 'created_at',
+      type: 'Date',
+      columnName: 'created_at',
+      isRequired: false,
+      defaultValue: null,
+      isReadOnly: false,
+      isSortable: true,
+      isFilterable: true,
+      isVirtual: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      validations: [],
+      integration: null,
+    },
+    {
+      field: 'updated_at',
+      type: 'Date',
+      columnName: 'updated_at',
+      defaultValue: null,
+      isRequired: false,
+      isReadOnly: false,
+      isSortable: true,
+      isFilterable: true,
+      isVirtual: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      validations: [],
+      integration: null,
+    },
+    {
+      field: 'deleted_at',
+      type: 'Date',
+      columnName: 'deleted_at',
+      defaultValue: null,
+      isRequired: false,
+      isReadOnly: false,
+      isSortable: true,
+      isFilterable: true,
+      isVirtual: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      validations: [],
+      integration: null,
+    },
+    {
+      field: 'name',
+      type: 'String',
+      columnName: 'name',
+      defaultValue: null,
+      isRequired: false,
+      isReadOnly: false,
+      isSortable: true,
+      isFilterable: true,
+      isVirtual: false,
+      description: null,
+      reference: null,
+      inverseOf: null,
+      relationships: null,
+      enums: null,
+      validations: [],
+      integration: null,
+    },
+  ],
```

Comment:
[Code Style] The lighter the fixtures the easier we read and maintain it.
I would reduce the number of declared fields.

All the fields that are not necessary for the test purposes should be deleted IMO,

Reply:
:+1: I removed the `updatedAt`, `createdAt` and `deletedAt`